app/services/romm.py

```
@@ -0,0 +1,441 @@
+
+    Resolved through :mod:`services.romm_settings`, so a token saved in the app
+    wins over the ``ROMM_TOKEN`` environment default and takes effect without a
+    restart. Deliberately not a field on the global ``Settings`` object: a
+    secret there leaks into every ``repr()`` and config dump.
+    """
+    from services import romm_settings
+
+    return romm_settings.token()
+
+
+def _urlopen(req: urllib.request.Request, origin_url: str):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_urlopen`

(ANN202)